tests/php/test-fieldmanager-richtextarea-field.php

```diff
@@ -565,4 +580,43 @@ public function test_customize_buttons_filter() {
 		$value = rand_str();
 		$this->assertEquals( $value, $fm->customize_buttons( $value ) );
 	}
+
+	/**
+	 * Test to confirm that not defining a default value for a RichTextArea field
+	 * does not cause that field to throw a deprecation warning when a RichTextArea
+	 * is used.
+	 *
+	 * @see https://github.com/alleyinteractive/wordpress-fieldmanager/issues/864
+	 */
+	public function test_default_value_does_not_throw_deprecation() {
+		set_error_handler( // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_set_error_handler
+			/**
+			 * Convert deprecations to exceptions. This was removed from PHPUnit in
+			 * PHPUnit 10, so doing so manually here for future compatibility.
+			 *
+			 * @see https://github.com/sebastianbergmann/phpunit/issues/5062
+			 */
+			function ( $errno, $errstr ) {
+				throw new Fieldmanager_Deprecation_Exception( $errstr, $errno );
```

Comment:
Rather than throwing, you could keep track of the number of deprecation messages in a variable in this class and then run an assertion on it later. Tests should have assertions, and that would be a way to bring an assertion into this test rather than relying on it not throwing an exception.